packages/api/src/utils/router.js

```diff
@@ -167,7 +169,10 @@ class Router {
       rsp = this.options.onNotFound(req)
     }
 
-    return postHandlers.reduce((r, handler) => handler(req, r), rsp)
+    const out = postHandlers.reduce((r, handler) => handler(req, r), rsp)
+
+    ctx.log.timeEnd('request')
+    return ctx.log.end(out)
```

Comment:
What happens if this fails? Can the logging not be on the critical path - i.e. in a `waitUntil`?

Reply:
ah its not so bad, `log.end` just adds the `Server-Timing` header and returns the response the logging is done inside `waitUntil` so i think its ok to keep this as is